pkg/app/kafka.go

```diff
@@ -163,6 +163,126 @@ func (kc *KafkaCluster) Install(ctx context.Context, namespace string) error {
 	return nil
 }
 
+func (kc *KafkaCluster) GetClusterScopedResources(ctx context.Context) []crv1alpha1.ObjectReference {
+	return []crv1alpha1.ObjectReference{
```

Comment:
Let's comment and split the resource by group
```suggestion
	return []crv1alpha1.ObjectReference{
	// ClusterRoles
```